### ISSUE TITLE
Ethereum node connection error

### ISSUE THREAD

**Issue by itredd** [ethereum/mist#967]
i ran 0.7.6,  7 days ago with no problems til this day, i got ethereum node connection error, when starting
0.8.0, or 0.7.6 wont start with the same problem, i got win7 64, i got my chain running last 7 days ago, appreciate for any help

**Comment by luclu** [comment]
@itredd please follow the guidelines.
> Node type: geth/eth
OS: linux/windows/osx
Network type: main/test
Mist version: 0.0.0

Please check the already existing issues, before creating an issue. We would like to keep duplicates at a minumim.
Read the https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide first, where we summarized a few of the known issues.

Add the following to issue, when possible:

- Screenshots
- Check the console, of Mist (CTRL/CMD + ALT + i) and take a screenshot.
- Log files (as ZIP archive)
-   osx: ~/Library/Application Support/Mist/node.log (Open Home folder > Show View Options > Show Library Folder)
-   windows: %APPDATA%/Roaming/Mist/node.log
-   linux: ~/.config/Mist/node.log

**Comment by itredd** [comment]
![ethereum](https://cloud.githubusercontent.com/assets/20442725/16812397/2bca2d7e-492d-11e6-9fd0-d1cde2298fc1.jpg)

**Comment by itredd** [comment]
ZIP FILES WONT LOG

**Comment by luclu** [comment]
Please rename the `%appdata%\Ethereum\Mist` folder and try to restart.

**Comment by itredd** [comment]
???
![mist](https://cloud.githubusercontent.com/assets/20442725/16816211/4ee42d36-493d-11e6-99cf-8e1d46f19689.jpg)

**Comment by luclu** [comment]
Let's keep it simple. Backup the `keystore` folder and delete both the `Ethereum` and `Mist` folder.

**Comment by itredd** [comment]
Yes thanks, i got it loading now

Sent from my iPhone

> On 13 Jul 2016, at 22:09, Luca Zeug <notifications@github.com> wrote:
> 
> Let's keep it simple. Backup the keystore folder and delete both the Ethereum and Mist folder.
> 
> -
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

____________________________________________________________
FREE 3D EARTH SCREENSAVER - Watch the Earth right on your desktop!
Check it out at http://www.inbox.com/earth

**Comment by itredd** [comment]
i got my botrh walletrs, but daos are gone

**Comment by luclu** [comment]
Great. Now you have to re-add the "The DAO" token to get them displayed.

**Comment by jp3492** [comment]
I am able to run the ethereum wallet but when im trying to add an account in the interface, my cmd shows me this connection error message. How can i solve this issue?
![screenshot 160](https://user-images.githubusercontent.com/8927423/27123334-d0dd44fe-50ed-11e7-8238-c6788b8e905e.png)

**Comment by evertonfraga** [comment]
@jp3492 please try to start Geth separately. Here's how.

Make sure to have Mist closed. then:
1. download latest version from http://geth.ethereum.org/downloads/
2. on your cmd.exe, type `geth --fast --cache 2048`
3. open Mist

**Comment by curator999** [comment]
Hi I have Mac OS X and have same problem. I have both Ethereum and Mist 0.9.0. Can you explain how to use your advice on mac? 
_"backup the keystore and chaindata folder
delete both the Ethereum and Mist folder
restore the folders from 1"._
I don't have folder that calling Mist and don't have folder calling Ethereum. My mist inside the folder Ethereum. So how do I deal with that? And what that mean - restore the folder from 1 (from the keystone and chain data folder. Please advice!

**Comment by rivaderci88** [comment]
hi there, sorry if I was unclear.  My issue is with the Application that i downloaded called "ethereum wallet".  I downloaded it using chrome from github.  Now a few months later  i can't reopen the application successfully, i just get the following message.  Is this something you would be able to help me with?

If not is there a way I can use myetherwallet.com to send my coins to another wallet without having the private key?  maybe the Json file or something on my computer that would be helpful?

All i know is I used to open this application without an issue, and now I can't, and thus cannot access my ether.



  Inline image 1
